### ISSUE TITLE
Port `receive_body_as_binary` option to Elixir

### ISSUE THREAD

**Issue by robertoaloi** [aws-beam/aws-codegen#71]
In #67 we introduced a `receive_body_as_binary` option which is currently only available in the generated Erlang code.
Ensure the option is ported to the generated Elixir code, too.

Hints:

```
https://github.com/aws-beam/aws-elixir/blob/master/lib/aws/request.ex#L44
https://github.com/aws-beam/aws-elixir/blob/master/lib/aws/request.ex#L104
```

**Comment by philss** [comment]
I'm on it :)

**Comment by philss** [comment]
Fixed by #74 and https://github.com/aws-beam/aws-elixir/pull/61. Thank you!
